src/java.desktop/share/classes/javax/swing/text/html/StyleSheet.java

```diff
@@ -1014,12 +1014,35 @@ public float getPointSize(String size) {
     }
 
     /**
-     * Converts a color string such as "RED" or "#NNNNNN" to a Color.
-     * Note: This will only convert the HTML3.2 color strings
-     *       or a string of length 7;
-     *       otherwise, it will return null.
+     * Converts a color string such as "RED",  "rgb(r g b)", "rgb(r g b a)",
+     * "rgba(r g b a)" or "#NNN", "#NNNN", "#NNNNNN",
+     * "#NNNNNNNN" to a Color.
+     * <p>
+     * Note: This will only convert strings which use any of the following:
+     * <ul>
+     *   <li><a href="https://www.w3.org/TR/css-color-4/#named-colors">named colors</a>,</li>
+     *   <li><a href="https://www.w3.org/TR/css-color-4/#hex-notation">hex color notation</a>
+     * starting with {@code #} followed by 3, 4, 6, or 8 hexadecimal digits,</li>
+     *   <li><a href="https://www.w3.org/TR/css-color-4/#rgb-functions">`rgb()` and `rgba()`
+     * functions</a></li>
+     * </ul>
+     * as specified by the <a href="https://www.w3.org/TR/css-color-4/">CSS Color Module Level 4</a>.
```

Comment:
Since this PR : https://github.com/openjdk/jdk/pull/13360/files
We have the new {@spec .. } tag whereever we reference an external spec.
So you should add it here too.

Reply:
Also, I think that in the public class doc of CSS.java which talks about CSS1, you need to add a comment
"Additionally, the color strings from CSS 4 are supported" and again point to the spec.